src/test/java/org/takes/misc/VerboseIterableTest.java

```diff
@@ -0,0 +1,32 @@
+package org.takes.misc;
+
+import java.util.Arrays;
+import java.util.List;
+
```

Comment:
@m4solutions Add a comment here saying `Tests for {@link VerboseIterable}.` followed by a blank comment line.
```java
/**
 * Tests for {@link VerboseIterable}.
 * 
 * @author etc...
```